<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassiveAcquisitionWidget</class>
 <widget class="QDialog" name="PassiveAcquisitionWidget">
  <property name="windowTitle">
   <string>Passive Data Acquisition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Server Connection (required)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="tomviz::ConnectionsWidget" name="connectionsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="vBottomSpacer0">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="watchPathLabel">
     <property name="text">
      <string>Watch Path (required)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="watchPathLineEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="formatTabWidget">
     <widget class="QWidget" name="basicTab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="basicTabLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="fileNameFormatLabel">
         <property name="text">
          <string>File Name Format</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="fileFormatCombo">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="customGroupBox">
         <property name="title">
          <string>Custom format</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_group">
          <item>
           <widget class="tomviz::CustomFormatWidget" name="customFormatWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="vBottomSpacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="tomviz::AdvancedFormatWidget" name="advancedTab" native="true">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="vBottomSpacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="testFormatLabel">
     <property name="text">
      <string>Sample filename</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="testFileFormatEdit">
     <property name="placeholderText">
      <string>Paste a sample filename</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTableWidget" name="testTableWidget"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QWidget" name="testTablePlaceholder" native="true"/>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="vBottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopWatchingButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Watching</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="watchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Watch</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::ConnectionsWidget</class>
   <extends>QWidget</extends>
   <header>ConnectionsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::CustomFormatWidget</class>
   <extends>QWidget</extends>
   <header>CustomFormatWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::AdvancedFormatWidget</class>
   <extends>QWidget</extends>
   <header>AdvancedFormatWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
